data/common.yaml

```diff
@@ -42,6 +42,19 @@ profile_lustre::service::lnet_service_enabled: false
 profile_lustre::service::lnet_service_name: "lnet"
 profile_lustre::service::lnet_service_running: false
 
+profile_lustre::telegraf::lnet_router_stats::enabled: true
+profile_lustre::telegraf::lnet_router_stats::script_cfg:
+  # Array of Mount Paths eg. "/taiga"
```

Comment:
The example shows "/taiga" but holli-control uses "taiga" (no preceding /). And the README.md doesn't use the /. Same for Magnus RHEL 7 (not Puppetized). Seems like it's OK without the /, so can we update this comment (example) to avoid confusion?

Reply:
Fair enough! i fixed it to mirror the README.md: https://github.com/ncsa/puppet-profile_lustre/pull/10/commits/c66bcb776ebf93555d60bdfe75ee73c90129f0e8

Additional commits will be squashed before merging